ssr-playground/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "private": true,
+    "name": "@lwc/ssr-playground",
+    "version": "3.0.0",
+    "type": "module",
+    "description": "Playground project to experiment with LWC.",
+    "scripts": {
+        "dev": "node ./server.js",
+        "csr-dev": "rollup -c --watch",
+        "ssr:playground": "cd .. && yarn run build && cd ssr-playground && yarn run dev"
+    },
+    "dependencies": {
+        "@lwc/engine-server": "^3.0.0",
+        "@lwc/rollup-plugin": "^3.0.0",
+        "@rollup/plugin-alias": "^4.0.2",
+        "@rollup/plugin-replace": "^5.0.2",
+        "express": "^4.18.2",
+        "html-entities": "^2.3.3",
+        "lwc": "^3.0.0",
```

Comment:
These should be exact versions instead of `^` versions so that Lerna can link them up correctly, and so we know exactly what version someone has when they file repros. This is how we do it in the other playground:

https://github.com/salesforce/lwc/blob/5799575bf2ad91a6d4e4491e2fc6ab6ac8a317bd/playground/package.json#L12